pkg/relayer/proxy/server_builder.go

```diff
@@ -137,6 +145,14 @@ func (rp *relayerProxy) waitForSupplierToStake(
 				supplierAddress,
 			)
 			time.Sleep(supplierStakeWaitTime * time.Second)
+
+			timeElapsed := time.Since(startTime)
+			fmt.Println(timeElapsed, supplierMaxStakeWaitTime*time.Second, timeElapsed > supplierMaxStakeWaitTime*time.Second)
+			if timeElapsed > supplierMaxStakeWaitTime*time.Second {
+				panic(fmt.Sprintf("Waited too long (%d seconds) for the supplier to stake. Exiting...", supplierMaxStakeWaitTime))
```

Comment:
Can you elaborate on the rationale here? IIRC the original intention of the relayminer waiting for it's account to stake was so that provisioned relayminers can be running and ready to participate in the load test when the time comes. I think this would introduce friction regarding the timing of starting localnet vs. starting the load test.

Reply:
Going to #PUC and resolve. Lmk if we should keep discussing.